doc/INSTALL.md

```diff
@@ -52,8 +52,8 @@ production workload and a `dev` installation for experimentation, the below comm
 two dataplanes, one into the `pod` and another one into the `dev` namespace.
 
 ```console
-helm install stunner stunner/stunner-prod --create-namespace --namespace=stunner-prod
-helm install stunner stunner/stunner-dev --create-namespace --namespace=stunner-dev
+helm install stunner stunner/stunner --create-namespace --namespace=stunner-prod
+helm install stunner stunner/stunner --create-namespace --namespace=stunner-dev
```

Comment:
Same thing, I'd suggest `helm install stunner-dev stunner/stunner --create-namespace --namespace=stunner-dev`

Reply:
yes!

Reply:
Same as for the other comment. There is no stunner/stunner-dev chart available. Changes are fine though.